mkdocs.yml

```
@@ -20,44 +20,53 @@ google_analytics:
 nav:
   - 简介:
     - Getting Started: index.md
-    - OI 赛事与赛制: intro/oi.md
-    - ICPC/CCPC 赛事与赛制: intro/icpc.md
-    - 学习路线: intro/roadmap.md
-    - 学习资源: intro/resources.md
-    - 常见错误: intro/common-mistakes.md
-    - 常见技巧: intro/common-tricks.md
-    - 非传统题: intro/non-trad.md
-    - 工具软件:
-      - 评测工具: intro/judgers.md
-      - WSL (Windows 10): intro/wsl.md
-      - Special Judge: intro/spj.md
-      - Testlib:
-        - Testlib 简介: intro/testlib/index.md
-        - 通用: intro/testlib/general.md
-        - Generator: intro/testlib/generator.md
-        - Validator: intro/testlib/validator.md
-        - Interactor: intro/testlib/interactor.md
-        - Checker: intro/testlib/checker.md
-      - Polygon: intro/polygon.md
-      - OJ 工具: intro/oj-tool.md
-      - LaTeX 入门: intro/latex.md
-    - Docker 部署: intro/docker-deploy.md
     - 关于本项目: intro/about.md
     - 如何参与: intro/htc.md
     - 格式手册: intro/format.md
     - F.A.Q.: intro/faq.md
+    - 用 Docker 部署 OI Wiki: intro/docker-deploy.md
     - 致谢: intro/thanks.md
+  - 比赛相关:
+    - 赛制:
```

```suggestion
    - 赛事:
```

感觉赛制更像是 OI 赛制 / ACM 赛制 / IOI 赛制 / CF 赛制 / ACM + open hacking 赛制 / UOJ 赛制 / AtCoder 赛制 / 洛谷乐多赛 这种。

---

有理